src/Agent.Sdk/SecretMasking/ISecret.cs

```diff
@@ -8,5 +8,5 @@ internal interface ISecret
     /// <summary>
     /// Returns one item (start, length) for each match found in the input string.
     /// </summary>
-    IEnumerable<ReplacementPosition> GetPositions(String input);
+    IEnumerable<Replacement> GetReplacements(String input);
```

Comment:
>Replacement [](http://example.com/codeflow?start=16&length=11)

`ReplacementPositions` now has additional metadata, for example, the preferred token to use for redaction, so I performed a rename.